File: core/tmp/Java/src/com/dbc/algorithm/Math/BulbSwitcher.java
package com.dbc.algorithm.Math;

public class BulbSwitcher {
    public static int bulbSwitch(int n) {
        int result = 0;
        for (int i = 1; i < n + 1; i++) {
            if (i * i <= n)
                result++;
            else break;
        }
        return result;
    }
}

File: core/tmp/Python/math/bulb_switcher.py
def bulbSwitch(n: int) -> int:
    result = 0
    for i in range(1, n + 1):
        if i * i <= n:
            result += 1
        else:
            break
    return result

if __name__ == "__main__":
    print(bulbSwitch(100000000))
